bitrise.yml

```diff
@@ -13,15 +13,16 @@ stages:
   stage_2:
     workflows:
     - build_and_test_1_SmokeTest2: {}
-    - build_and_test_2_SmokeTest: {}
-    - build_and_test_3_SmokeTest3: {}
-    - build_and_test_4_SmokeTest4: {}
-    - build_and_test_5_UnitTest: {}
-    - just_build_fennec_to_get_warnings: {}
+    #- build_and_test_2_SmokeTest: {}
+    #- build_and_test_3_SmokeTest3: {}
+    #- build_and_test_4_SmokeTest4: {}
+    #- build_and_test_5_UnitTest: {}
+    #- just_build_fennec_to_get_warnings: {}
 workflows:
   configure_build:
     before_run:
     - 1_git_clone_and_post_clone
+    - Decision_to_run_UI_Tests
```

Comment:
We can remove this from here, added to see if the scripts was working in the config step. We can't save and share the env variable value from different stages and workflows that's why we would need to run the workflow within each test workflow and not here